ansible_navigator/configuration/application_configuration.py

```diff
@@ -0,0 +1,199 @@
+    else:
+        command = "vi +{line_number} {filename}"
+    return command
+
+
+PLUGIN_TYPES = (
+    "become",
+    "cache",
+    "callback",
+    "cliconf",
+    "connection",
+    "httpapi",
+    "inventory",
+    "lookup",
+    "module",
+    "netconf",
+    "shell",
+    "strategy",
+    "vars",
+)
+
+ApplicationConfiguration = Config(
+    application_name="ansible-navigator",
+    post_processor=ApplicationPostProcessor(),
+    subcommands=[
+        SubCommand(name="collections", description="Explore available collections"),
+        SubCommand(name="config", description="Explore the current ansible configuration"),
+        SubCommand(name="doc", description="Review documentation for a module or plugin"),
+        SubCommand(name="inventory", description="Explore an inventory"),
+        SubCommand(name="load", description="Explore a playbook artifact"),
+        SubCommand(name="run", description="Run a playbook"),
+    ],
+    entries=[
+        Entry(
+            name="app",
+            description="Placeholder for subcommand name",
+            subcommand_value=True,
+            value=EntryValue(default="welcome"),
+        ),
+        Entry(
+            name="cmdline",
+            description="Placeholder for argparse remainder",
+            value=EntryValue(default=[]),
+        ),
+        Entry(
+            name="container_engine",
+            choices=["podman", "docker"],
+            cli_parameters=CliParameters(short="--ce"),
+            description="Specify the container engine",
+            value=EntryValue(default="podman"),
+        ),
+        Entry(
+            name="editor_command",
+            cli_parameters=CliParameters(short="--ecmd"),
+            description="Specify the editor comamnd",
+            settings_file_path_override="editor.command",
+            value=EntryValue(default=generate_editor_command()),
+        ),
+        Entry(
+            name="editor_console",
+            choices=[True, False],
+            cli_parameters=CliParameters(short="--econ"),
+            description="Specify if the editor is console based",
+            settings_file_path_override="editor.console",
+            value=EntryValue(default=True),
+        ),
+        Entry(
+            name="execution_environment",
+            choices=[True, False],
+            cli_parameters=CliParameters(short="--ee"),
+            description="Enable the use of an execution environment",
+            value=EntryValue(default=True),
+        ),
+        Entry(
+            name="execution_environment_image",
+            cli_parameters=CliParameters(short="--eei"),
+            description="The name of the execution environment image",
+            value=EntryValue(default="image_here"),
+        ),
+        Entry(
+            name="inventory",
+            cli_parameters=CliParameters(action="append", nargs="+", short="-i"),
+            description="Specify an inventory path",
```

Comment:
In addition to inventory path the `ansible-inventory -i` option can take  comma-separated host list as input

Reply:
will add to description